Production/govuk_ios/Views/Home/WidgetView.swift

```diff
@@ -2,34 +2,25 @@ import UIKit
 import Foundation
 import UIComponents
 
-class HomeWidgetView: UIView {
+protocol WidgetInterface {
+    var viewModel: HomeWidgetViewModel { get }
+}
+
+class WidgetView: UIView {
     private lazy var stackView: UIStackView = {
-        let localView = UIStackView()
+        let localView = (widget as? UIStackView)!
```

Comment:
This feels a little dangerous. Also the `as?` is undone by the `!` you could just do `widget as! UIStackView`